src/vulkan-renderer/wrapper/device.cpp

```diff
@@ -34,6 +32,95 @@ constexpr float default_queue_priority = 1.0f;
 
 namespace inexor::vulkan_renderer::wrapper {
 
+bool Device::device_extension_supported(const VkPhysicalDevice graphics_card, const std::string &extension_name) {
```

Comment:
Just see it now, the methods are not ordered like in th hpp.

Reply:
Now it is for device.

Reply:
Done.